components/alert/alertPage.css

```diff
@@ -2,13 +2,18 @@
     display: flex;
     flex-direction: column;
     align-items: center;
+    padding-left: 20rem;
+    padding-right: 20rem;
```

Comment:
padding : 0 20rem;  this might look better. Again personal choice.

Reply:
+1

Reply:
I am following the explicit way site wide, unless all 4 require change.
But yes I agree with you, it's just to maintain consistency.